app/assets/javascripts/factories/MimeDictionary.coffee

```diff
@@ -0,0 +1,170 @@
+    'application/xml': 'xml document'
+    'application/example': 'common file'
+    # vendor-specific / common stuff
+    'application/vnd.debian.binary-package': 'debian package'
+    'application/vnd.google-earth.kml+xml': 'google earth kml file'
+    'application/vnd.google-earth.kmz': 'google earth kmz file'
+    'application/vnd.android.package-archive': 'android pkg file'
+    'application/vnd.ms-xpsdocument': 'xps document'
+    'text/vnd.abc': 'abc music file'
+    'application/x-chrome-extension': 'chrome extension'
+    'application/x-dvi': 'dvi file'
+    'application/x-font-ttf': 'TrueType font file'
+    'application/x-javascript': 'javascript'
+    'application/x-latex': 'latex file'
+    'application/x-mpegURL': 'playlist file'
+    'application/x-shockwave-flash': 'flash file'
+    'application/x-www-form-urlencoded': 'form data'
+    'application/x-xpinstall': 'mozilla addon'
+    # audio
+    'audio/example': 'audio file'
+    'audio/basic': 'μ-law audio'
+    'audio/L24': 'audio file'
+    'audio/mp4': 'mp4 audio'
+    'audio/mpeg': 'mpeg file'
+    'audio/ogg': 'ogg vorbis audio'
+    'audio/flac': 'native flac audio'
+    'audio/opus': 'opus audio'
+    'audio/vorbis': 'vorbis audio'
+    'audio/vnd.rn-realaudio': 'RealPlayer audio'
+    'audio/vnd.wave': 'wav file'
+    'audio/webm': 'webm audio'
+    'audio/x-aac': 'aac audio'
+    'audio/x-caf': 'apple caf audio'
+    'image/x-xcf': 'gimp image'
+    # compressed folders
+    'application/zip': 'compressed folder'
+    'application/x-zip-compressed': 'compressed folder'
+    'application/x-compress': 'compressed folder'
+    'multipart/x-zip': 'compressed folder'
+    'application/gzip': 'compressed folder'
+    'application/x-stuffit': 'compressed archive'
+    'application/x-tar': 'tarball'
+    'application/x-rar-compressed': 'rar compressed folder'
+    'application/x-7z-compressed': '7-zip compressed file'
+    # documents
+    'application/pdf': 'pdf document'
+    'application/vnd.oasis.opendocument.text': 'opendocument text'
+    'application/vnd.oasis.opendocument.spreadsheet': 'opendocument spreadsheet'
+    'application/vnd.oasis.opendocument.presentation': 'opendocument presentation'
+    'application/vnd.oasis.opendocument.graphics': 'opendocument graphics'
+    'application/vnd.ms-excel': 'excel spreadsheet'
+    'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet': 'excel spreadsheet'
+    'application/vnd.ms-powerpoint': 'powerpoint presentation'
+    'application/vnd.openxmlformats-officedocument.presentationml.presentation': 'powerpoint presentation'
+    'application/vnd.ms-word': 'word document'
+    'application/vnd.openxmlformats-officedocument.wordprocessingml.document': 'word document'
+    'application/x-iwork-keynote-sffkey': 'keynote presentation'
+    'application/vnd.google-apps.document': 'google drive document'
+    'application/vnd.google-apps.presentation': 'google drive presentation'
```

Comment:
These are the default apps in google drive that we probably should support:
'application/vnd.google-apps.spreadsheet': google drive spreadsheet'
'application/vnd.google-apps.map': 'google drive map'
'application/vnd.google-apps.drawing': 'google drive drawing'
'application/vnd.google-apps.form': 'google drive form'
'application/vnd.google-apps.spreadsheet': 'google drive spreadsheet'
'application/vnd.google-apps.fusiontable': 'google drive fusiontable'